core/pom.xml

```diff
@@ -75,6 +75,25 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!-- Annotations for the CRD generator -->
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
We've not aligned on a proper way to do validation, so this needs to be removed, or we need a separate discussion around this.

Reply:
removed